src/components/v-mapbox-layer.js

```diff
@@ -1,44 +1,77 @@
 export default {
   name: 'v-mapbox-layer',
-  render () { },
-  data () {
-    return {
-    };
-  },
+
+  inject: ['getMap'],
+
+  render: () => null,
+
   props: {
-    options: {
-      default: () => {
-        return {};
-      },
-      type: [Object, String]
+    layer: {
+      type: Object,
+      default: () => ({})
     },
-    // allows to place a layer before another
+
+    // Allows to place a layer before another
     before: {
       type: String,
-      required: false
+      default: null
```

Comment:
before is an [optional](https://docs.mapbox.com/mapbox-gl-js/api/map/#map-instance-members) argument, so I think it should be passed as `undefined` rather than `null`